### PR TITLE
Re-added Channel Separation Example

### PR DESCRIPTION
**Pull request by madelinecoleman1**

### CODE REVIEW

File: examples/07-EMIT/ComputeChannelSeparation.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+import numpy as np
+
+###############################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode. ``"PYAEDT_NON_GRAPHICAL"``` is needed to generate
+# documentation only.
+# You can set ``non_graphical`` either to ``True`` or ``False``.
+# The ``NewThread`` Boolean variable defines whether to create a new instance
+# of AEDT or try to connect to existing instance of it if one is available.
```
Comment (PipKat) [inline examples/07-EMIT/ComputeChannelSeparation.py:52]:
```suggestion
# of AEDT or try to connect to existing instance of AEDT if one is available.
```

File: examples/07-EMIT/ComputeChannelSeparation.py [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+    
+###############################################################################
+# Load the results set
+# ~~~~~~~~~~~~~~~~~~~~
+# Create a results revision and load it for analysis.
+
+rev = emitapp.results.analyze()
+
+def get_rx_bands(rx_radio):
+    """Return a list of all Rx bands in a given radio.
```
Comment (PipKat) [inline examples/07-EMIT/ComputeChannelSeparation.py:107]: This docstrings are not in numpydoc format. Given that there is a "Returns" section, use "Get as the verb for this method and those that follow.

File: examples/07-EMIT/ComputeChannelSeparation.py [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+                or fuzzy_in_range(rx_start, tx_start, tx_stop)
+                or fuzzy_in_range(rx_stop, tx_start, tx_stop)
+            ):
+                overlapping.append((tx_radio, tx_band))
+    return overlapping
+
+###############################################################################
+# Iterate over all the receivers
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Iterates over each of the receivers in the project and finds any transmit
```
Comment (PipKat) [inline examples/07-EMIT/ComputeChannelSeparation.py:158]:
```suggestion
# Iterate over each of the receivers in the project and find any transmit
```

File: examples/07-EMIT/ComputeChannelSeparation.py [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+            '    Rx band "{}" has the following overlapping Tx bands:'.format(rx_band[1])
+        )
+        for tx_band in tx_bands:
+            overlapping.append((rx_band, tx_band))
+            print('        {}'.format(tx_band))
+
+###############################################################################
+# Print a list of overlapping bands
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Prints a list of overlapping receivers and bands.
```
Comment (PipKat) [inline examples/07-EMIT/ComputeChannelSeparation.py:178]:
```suggestion
# Print a list of overlapping receivers and bands.
```

File: examples/07-EMIT/ComputeChannelSeparation.py [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+    """
+    ret = []
+    for a in a_list:
+        if a not in ret:
+            ret.append(a)
+    return ret
+
+def show_separation_table(separation_results, title='In-band Separation (MHz)'):
+    """Create a scenario matrix-like table to display the maximum
+    channel separate required for each transmit/receive band combination.
```
Comment (PipKat) [inline examples/07-EMIT/ComputeChannelSeparation.py:303]: separation rather than separate?

**Review by jsalant22 — APPROVED** [review]

**Pull request merged**